README.md

```diff
@@ -146,6 +146,10 @@ Most aspects of your cluster setup can be customized with environment variables.
 
    Defaults to `false`.
 
+ - **USE_METRICS_SERVER** defines whether to deploy or not the [Kubernetes Metrics Server](https://github.com/kubernetes-incubator/metrics-server)
+
+   Defaults to `false`.
```

Comment:
```suggestion
   Defaults to `true`.
```

This is required for the [Horizontal Pod Autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) to work.

Reply:
I would say agree because HPA is my motivation for adding this feature. The reason why I put it to false by default is that I haven't tested it in Windows nor RBAC (and I probably won't because I don't have this kind of setup).

Reply:
I'm sure @bmcstdio can help here, right Bruno?

Reply:
Definitely. However, it seems to me that this PR needs some love. I checked out this branch and ran the following command:

```
$ NODES=1 USE_METRICS_SERVER=true vagrant up
```

Turns out the Kubernetes API can never be reachable:

```
$ kubectl get node
Unable to connect to the server: net/http: TLS handshake timeout
```

This does not happen in `USE_METRICS_SERVER` is set to `false`.

I also noticed high load on the system - far higher than usual. While I am not sure, part of what the root cause for this may be the fact that the following flag is not being defined:

```
--enable-aggregator-routing=true
```

https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/#enable-apiserver-flags

Reply:
Yeah, definitely need more tests. I only tested it on macOS. Also, good catch on `--enable-aggregator-routing=true`.